src/qtwebkit-1-fixes.patch

```diff
@@ -2,94 +2,221 @@ This file is part of MXE. See LICENSE.md for licensing information.
- for(binding, IDL_BINDINGS) {
-     # We need "$$binding" instead of "$$binding ", because Windows' echo writes trailing whitespaces. (http://wkb.ug/88304)
+--- a/Source/cmake/FindICU.cmake.orig	2017-02-07 21:39:31.536985800 +0300
++++ b/Source/cmake/FindICU.cmake	2017-02-07 21:43:45.849531600 +0300
+@@ -49,7 +49,7 @@
+     pkg_check_modules(PC_ICU_I18N icu-i18n)
+     find_library(
+         ICU_I18N_LIBRARY
+-        NAMES icui18n icuin cygicuin cygicuin32
++        NAMES icuin icui18n cygicuin cygicuin32
+         HINTS ${PC_ICU_I18N_LIBRARY_DIRS}
+               ${PC_ICU_I18N_LIBDIR}
+         DOC "Libraries to link against for ICU internationalization")
 
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Martchus <martchus@gmx.net>
-Date: Sun, 25 Sep 2016 22:21:54 +0200
-Subject: [PATCH 2/3] Include intrin.h for declaration of _mm_mfence
+--- qtwebkit-tp5/Source/WebCore/platform/network/create-http-header-name-table.orig	2017-02-07 23:28:00.520278300 +0300
++++ qtwebkit-tp5/Source/WebCore/platform/network/create-http-header-name-table	2017-02-07 23:29:15.412561900 +0300
+@@ -54,7 +55,7 @@
+ 
+ http_header_names.sort()
+ 
+-gperf_file = open('HTTPHeaderNames.gperf', 'w')
++gperf_file = open('HTTPHeaderNames.gperf', 'wb')
+ gperf_file.write('''
+ %{
+ /*
+--- qtwebkit-tp5/Source/WebCore/css/makeSelectorPseudoClassAndCompatibilityElementMap.py.orig	2017-02-08 00:23:27.256556700 +0300
++++ qtwebkit-tp5/Source/WebCore/css/makeSelectorPseudoClassAndCompatibilityElementMap.py	2017-02-08 00:23:33.040887600 +0300
+@@ -57,7 +57,7 @@
+ def expand_ifdef_condition(condition):
+     return condition.replace('(', '_').replace(')', '')
+ 
+-output_file = open('SelectorPseudoClassAndCompatibilityElementMap.gperf', 'w')
++output_file = open('SelectorPseudoClassAndCompatibilityElementMap.gperf', 'wb')
+ 
+ output_file.write("""
+ %{
+--- qtwebkit-tp5/Source/WebCore/css/makeSelectorPseudoElementsMap.py.orig	2017-02-08 00:23:46.797674400 +0300
++++ qtwebkit-tp5/Source/WebCore/css/makeSelectorPseudoElementsMap.py	2017-02-08 00:23:51.732956700 +0300
+@@ -57,7 +57,7 @@
+ def expand_ifdef_condition(condition):
+     return condition.replace('(', '_').replace(')', '')
+ 
+-output_file = open('SelectorPseudoElementTypeMap.gperf', 'w')
++output_file = open('SelectorPseudoElementTypeMap.gperf', 'wb')
+ 
+ output_file.write("""
+ %{
+
+--- qtwebkit-tp5/Source/WebCore/bindings/scripts/preprocess-idls.pl.orig	2017-02-08 07:25:12.036907400 +0300
++++ qtwebkit-tp5/Source/WebCore/bindings/scripts/preprocess-idls.pl	2017-02-08 07:32:03.926466100 +0300
+@@ -179,9 +179,9 @@
+ sub CygwinPathIfNeeded
+ {
+     my $path = shift;
+-    if ($path && $Config{osname} eq "cygwin") {
++    if ($path && ($Config{osname} eq "cygwin" || $Config{osname} eq "msys")) {
+         if (not $cygwinPathAdded) {
+-            $ENV{PATH} = "$ENV{PATH}:/cygdrive/c/cygwin/bin";
++            $ENV{PATH} = "$ENV{PATH}:/usr/bin";
+             $cygwinPathAdded = 1;
```

Comment:
For me the patch didn't apply due to a white space issue in this line. In fact the ORIGINAL file has a trailing white space but the patch file doesn't.